### PR TITLE
Add footer links

### PR DESCRIPTION
**Pull request by dimak1**
Link styles were applying, as per this [issue](https://github.com/zeit/next.js/issues/610), so had to fix by adding :global(a).

### CODE REVIEW

**Comment by dimak1** [comment]
> Can you add a page for each of the three links that just has the `DefaultLayout` and some `TODO` text?

**Comment by dimak1** [comment]
Closed by mistake

**Comment by dimak1** [comment]
@matthieu-foucault for these pages (Disclaimer, Privacy, Accessibility and Copyright) we don't need a session, right?

**Comment by matthieu-foucault** [comment]
@dimak1 Yes, those pages do not need a session

**Comment by dimak1** [comment]
@matthieu-foucault DefaultLayout requires session

**Comment by matthieu-foucault** [comment]
Yes. It does need a query with a session, as it will change what is rendered in the header, but if you set `needsSession` and `needsUser` to `false`, it will not redirect the user if there is no session (see the index page for an example)

File: app/components/Layout/Footer.tsx [hunk 1]
```diff
@@ -7,10 +8,16 @@ const Footer = () => {
         <div className="container">
           <ul>
             <li>
-              <a href=".">Home</a>
+              <Link href="/resources/disclaimer">Disclaimer</Link>
             </li>
             <li>
-              <a href=".">Contact Us</a>
```
Comment (matthieu-foucault) [inline app/components/Layout/Footer.tsx]: I don't think those should be removed. `Home` should be a link to `/`, and you can add another page for `Contact Us`

Comment (dimak1) [reply app/components/Layout/Footer.tsx]: Added links and Contact Us page.

**Pull request merged**